bindings/python/vlsir/primitives.py

```diff
@@ -0,0 +1,16 @@
+""" 
+# Primitive Definitions 
+
+Loaded from adjacent file `vlsir.primitives.pb.txt`
+"""
+
+from pathlib import Path
+from .circuit_pb2 import Package
+
+# Get the definition-file from this file's directory
+here = Path(__file__).parent.absolute()
+prims = here / "vlsir.primitives.pb.txt"
+
+# And parse it into a `circuit.Package`
+pkg = Package()
+(open(prims, "r").read(), pkg)
```

Comment:
Don't understand how this Package creation method works

Reply:
Yeah you're right, it doesn't work. 
Line 16 is supposed to say: 

```python 
text_format.Parse(open(prims, "r").read(), pkg)
```

And this begs for a better unit-test than the existing one: 

```python
def test_primitives1():
    from vlsir.primitives import pkg
    assert isinstance(pkg, vlsir.circuit.Package)
```

Which passes even without parsing the content.